registry/src/pages/admin.js

```diff
@@ -0,0 +1,376 @@
+import React, { useEffect, useState } from "react";
+import { Container } from "react-bootstrap";
+import {
+  MDBBtn,
+  MDBModal,
+  MDBModalDialog,
+  MDBModalContent,
+  MDBModalHeader,
+  MDBModalTitle,
+  MDBModalBody,
+  MDBModalFooter,
+  MDBIcon,
+} from "mdb-react-ui-kit";
+
+const AdminSection = () => {
+  const [deprecateModal, setdeprecateModal] = useState(false);
```

Comment:
These states should be stored inside reducers & actions can be written to perform any state change. Since, there are too many state variables it will become hard to maintain it here.